.changeset/empty-beers-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+This patch adds HarnessURLReader. It only supports readUrl for now. readTree and search will be implemented next.
```

Comment:
```suggestion
Added `HarnessURLReader` with `readUrl` support.
```

Reply:
fixed @vinzscam thanks for the review :)